frontend/public/index.html

```diff
@@ -3,8 +3,7 @@
 <head>
     <meta charset="utf-8"/>
     <link rel="shortcut icon" type="image/png" href="/static/logo/favicon_32px.png"/>
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
-          integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <link rel="stylesheet" href="%PUBLIC_URL%/bootstrap-4.4.1.min.css">
```

Comment:
Does webpack/workbox now pick this up as something to precache?

Also is this the best way to include bootstrap? I feel like there's supposed to be some better way where you `npm install` and then `import` something and it bundles everything just right.

Reply:
It's probably not the best way. I didn't look into the Webpack way, but I could

Reply:
Let's just keep this in mind for now.